tests/unit/test_io_zarr.py

```diff
@@ -877,9 +891,16 @@ def test_soft_link_group(self):
         foofile = FooFile(buckets=[foobucket], foo_link=foo1)
         with ZarrIO(self.paths[0], manager=get_foo_buildmanager(), mode='w') as write_io:
             write_io.write(foofile)
+        with open(self.paths[0]+"/.zattrs", 'r') as f:
+            print(f.readlines())
+
         with ZarrIO(self.paths[0], manager=get_foo_buildmanager(), mode='r') as read_io:
             with ZarrIO(self.paths[1], mode='w') as export_io:
-                export_io.export(src_io=read_io, write_args=dict(link_data=False))
+                with self.assertWarnsWith(UserWarning, "Could not determine source_object_id for "
+                                                       "builder with path: /buckets/bucket1/foo_holder/foo1"):
+                    export_io.export(
```

Comment:
Check if this warning is really expected to be raised (e.g., which is the case if the file is not using the ROOT_NAME as the name of  the root builder or has no object_id assigned). Otherwise we should not see this error, and we may need to do something different during export. Just something to double check.

Reply:
This should be resolved by #120 which names FooFile correctly as `root`